Core/Core.xcodeproj/project.pbxproj

```diff
@@ -59,6 +59,8 @@
 		3B5369E2219CDD3500D3F62E /* URLExtensions.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3B5369E1219CDD3500D3F62E /* URLExtensions.swift */; };
 		3B5369E4219D0FD000D3F62E /* URLExtensionsTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3B5369E3219D0FD000D3F62E /* URLExtensionsTests.swift */; };
 		3B5425FC21B074500009D069 /* UIViewControllerExtensions.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3B5425FB21B074500009D069 /* UIViewControllerExtensions.swift */; };
+		3B5542442369F3EF007D3CE0 /* AssignmentListRequestable.gql.swift in Sources */ = {isa = PBXBuildFile; fileRef = 3B5542432369F3EE007D3CE0 /* AssignmentListRequestable.gql.swift */; };
```

Comment:
Not sure I like the `.gql.swift` extension. 🤔 I think `APIXXRequestable.swift` is fine. GraphQL is more of an implementation detail. But I don't feel too strongly about it.

Reply:
No problem, i don't mind changing